From 91a66a59b6c1435120ff83e245d7719411294386 Mon Sep 17 00:00:00 2001
From: David Benjamin <davidben@google.com>
Date: Tue, 4 Nov 2025 11:24:42 -0500
Subject: [PATCH] Fix util/bazel-example under Bazel 9.0.0rc1

We need to explicitly import rules_cc. See
https://github.com/bazelbuild/bazel-central-registry/issues/6402

Change-Id: I648dd64fb2a6c21264dd9d664643ec76cd3a297a
Reviewed-on: https://boringssl-review.googlesource.com/c/boringssl/+/83567
Commit-Queue: David Benjamin <davidben@google.com>
Reviewed-by: Lily Chen <chlily@google.com>
Auto-Submit: David Benjamin <davidben@google.com>
Commit-Queue: Lily Chen <chlily@google.com>

--- util/bazel-example/BUILD.bazel
+++ util/bazel-example/BUILD.bazel
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+load("@rules_cc//cc:defs.bzl", "cc_binary")
+
 cc_binary(
     name = "example",
     srcs = ["example.cc"],
diff --git a/util/bazel-example/MODULE.bazel b/util/bazel-example/MODULE.bazel
index dda818d978..1f6e6888ea 100644
--- util/bazel-example/MODULE.bazel
+++ util/bazel-example/MODULE.bazel
@@ -14,6 +14,8 @@
 
 module(name = "bazel-example")
 
+bazel_dep(name = "rules_cc", version = "0.2.8")
+
 bazel_dep(name = "boringssl")
 local_path_override(
     module_name = "boringssl",
